runtime/crab/src/lib.rs

```diff
@@ -189,9 +194,6 @@ frame_benchmarking::define_benchmarks! {
 	[pallet_vesting, Vesting]
 	[pallet_session, SessionBench::<Runtime>]
 	[pallet_timestamp, Timestamp]
-	// Bridge
-	[pallet_bridge_grandpa, BridgePolkadotGrandpa]
-	[pallet_fee_market, DarwiniaFeeMarket]
 }
```

Comment:
Any particular reason for renaming `mod.rs` to `pallets.rs`? Look like mod is a better name

Reply:
Too many files in the `pallet` folder.

It's not clearly to check mod.rs.

Reply:
LGTM, after switching to your branch.